input/resources/au-erequesting-filler.xml

```diff
@@ -0,0 +1,309 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CapabilityStatement xmlns="http://hl7.org/fhir">
+  <id value="au-erequesting-filler"/>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
+    <valueInteger value="0"/>
+  </extension>
+  <url value="http://hl7.org.au/fhir/ereq/CapabilityStatement/au-erequesting-filler"/>
+  <name value="AUeRequestingFillerCapabilityStatement"/>
+  <title value="AU eRequesting Filler CapabilityStatement"/>
+  <status value="draft"/>
+  <description value="This CapabilityStatement describes the basic rules for the [AU eRequesting Filler actor](ActorDefinition-au-erequesting-actor-filler.html) that is  is a system that accepts diagnostic service requests and makes diagnostic service requests accessible. The complete list of FHIR profiles, RESTful operations, and search parameters supported by AU eRequesting fillers are defined in this CapabilityStatement."/>
```

Comment:
update description - filler is responsible for finding and retrieving diagnostic service requests so that service providers can fulfil them.

Reply:
fixed